TezosKit/Conseil/ConseilClient.swift

```diff
@@ -47,7 +46,60 @@ public class ConseilClient: AbstractClient {
     )
   }
 
+  /// Retrieve sent and received transactions from an account.
+  ///
+  /// - Parameters:
+  ///   - account: The account to query.
+  ///   - limit: The number of transactions to return, defaults to 100.
+  ///   - completion: A completion callback.
+  public func transactions(
+    from account: String,
+    limit: Int = 100,
```

Comment:
Maybe allow for an optional parameter to add more predicates?

Reply:
Not totally in love with this but I see why it might be useful.

The idea of predicates is also constrained internally to RPC objects, which exposes a new idea in the API.

If you wanted a custom query, you could easily bind one up with 
```
let client = ConseilClient(...)
let query = ConseilQuery.query(..)
let rpc = ConseilQueryRPC(query, apiKey, ...)
client.send(rpc) { // do something with result) }
```

You can likewise just subclass ConseilQueryRPC if you always wanted the same query and do: 
```
let client = ConseilClient(...)
let rpc = CustomQueryRPC(query, apiKey, ...)
client.send(rpc) { // do something with result) }
```

The only awkward part is that you can now send queries with different params than exist in the client. Maybe there's a nicer API here? WDYT?

---

Aside from the above, this particular method will leak that this is backed by two queries.  For a given predicate, which query to you apply it to? One or both?

Reply:
An elegantish refactor could be to hold:
- APIKey
- Network
- Platform

constant at the client level and remove them from being configurable on a per-RPC basis.

Reply:
This refactor will do it: https://github.com/keefertaylor/TezosKit/pull/64